provisioning-modules/module-parameters.md

```diff
@@ -14,7 +14,8 @@ The parameters also contains the settings from the product itself.
 
 | Variable Name | Description |
 |---|---|
-| serviceid | The unique ID of the service.<br />Database Field: tblhosting.id
+| userid | The unique ID of the account owning the service.<br />Database Field: tblclients.id<br />This variable is named based on pre-WHMCS 8 naming conventions. Under current conventions, it would likely be named accountid |
```

Comment:
Thank you for your contribution. I would like to accept your change however to avoid ambiguity, I think we should remove the sentence pertaining to how it would likely be named.

The reference to the database table and column should be sufficient.

If you could make this change, I would be happy to accept your PR.

Thanks again!

```suggestion
| userid | The unique ID of the account owning the service.<br />Database Field: tblclients.id<br />This variable is named based on pre-WHMCS 8 naming conventions. Under current conventions, it would likely be named accountid |
```